test/common_utils.py

```diff
@@ -13,11 +13,11 @@
 import __main__  # noqa: 401
 
 
-IN_CIRCLE_CI = os.getenv("CIRCLECI", False) == "true"
+IN_OSS_CI = any(os.getenv(var) == "true" for var in ["CIRCLECI", "GITHUB_ACTIONS"])
```

Comment:
One common convention for CI providers is to set the `CI=true` environment variable. We could use that here as well, but I have no idea if that interferes with Meta internal systems. Thus, to be safe, we are explicit about the CI providers here. Given that we probably don't change them that often, I think this should be fine.